code/src/main/java/com/google/sps/data/BusinessNumber.java

```diff
@@ -0,0 +1,36 @@
+package com.google.sps.data;
+
+import java.time.Instant;
+
+// Keep track of information needed to contact a business
+public class BusinessNumber implements ContactStatus{
+
+  String nameOfBusiness;
+  long phoneNumber;
```

Comment:
Let's use a string for phone number rather than a long. Strings seem less likely to have strange errors (what happens if there is a parenthesis in the text we get back, for example?)

Reply:
Okay I changed it. I think this is the only variable that uses a bad data type for what is being stored?

Reply:
Yep the rest look reasonable.